filer/admin/folderadmin.py

```diff
@@ -803,10 +799,7 @@ def _clean_destination(self, request, current_folder,
             raise PermissionDenied
         # don't allow selected folders to be copied/moved inside
         #   themselves or inside any of their descendants
-        mgr = Folder._tree_manager
-        destination_in_selected = mgr.get_queryset_descendants(
-            selected_folders, include_self=True
-        ).filter(id=destination.pk).exists()
+        destination_in_selected = Folder._tree_manager.get_queryset_descendants(selected_folders, include_self=True).filter(id=destination.pk).exists()
```

Comment:
I made a mistake during a merge. It is fixed now, thank you.

Reply:
I wanted to get rid of the intermediate variable, mgr.
However, after testing Filer with copy/move of more than one folder, I saw another error: "Maximum recursion depth exceeded". It seems to happen because mptt switched to using ".only" in [this queryset](https://github.com/django-mptt/django-mptt/blob/62b8892a3dd5cd70c299af9144e79674b4f71c11/mptt/managers.py#L133) combined with our use of "self.parent_id" in the Folder model's init method seem to trigger the infinite recursion.
I couldn't find a better solution than calling "get_descendants" on each folder instance and then doing the same filter and check as the code I replaced.

Reply:
These resources helped during investigation:
[infinite recursion when using .only() on a mptt model](https://github.com/django-mptt/django-mptt/issues/176)
[Django .only() causing maximum recursion depth exceeded error?](https://stackoverflow.com/questions/45547802/django-only-causing-maximum-recursion-depth-exceeded-error)